src/libraries/System.Net.Security/src/System/Net/Security/NegotiateAuthentication.cs

```diff
@@ -356,21 +356,24 @@ public NegotiateAuthenticationStatusCode UnwrapInPlace(Span<byte> input, out int
         /// Computes the message integrity check of a given message.
```

Comment:
nit: dedup `message`?
```suggestion
        /// Computes the integrity check of a given message.
```